lib/elixir/lib/kernel.ex

```diff
@@ -4998,6 +4998,56 @@ defmodule Kernel do
     Macro.escape(NaiveDateTime.from_iso8601!(string))
   end
 
+  @doc ~S"""
+  Handles the sigil `~U` to create a UTC `DateTime`.
+
+  The lower case `~u` variant does not exist as interpolation
+  and escape characters are not useful for datetime sigils.
```

Comment:
Minor nitpick

```suggestion
  and escape characters are not useful for date time sigils.
```

Reply:
fwiw, I followed ~N docs: https://github.com/elixir-lang/elixir/blob/c37725095cea47a9c7bf1c81cb772c54016af83e/lib/elixir/lib/kernel.ex#L5005 but yeah, if it's preferred I'm happy to change it in both places.

Reply:
:+1: for `:non_utc_offset`.